js/ui/popup.d.ts

```diff
@@ -55,13 +54,13 @@ export type InitializedEvent = InitializedEventInfo<dxPopup>;
 export type ShownEvent = EventInfo<dxPopup>;
 
 /** @public */
-export type ResizeEvent = NativeEventInfo<dxPopup> & ResizeInfo;
+export type ResizeEvent = EventInfo<dxPopup> & ResizeInfo;
```

Comment:
```suggestion
export type ResizeEvent = NativeEventInfo<dxPopup, MouseEvent | TouchEvent> & ResizeInfo;
```

`EventInfo` and `ResizeInfo` does not contains the `event` field